packs/data/familiar-abilities.db/skilled.json

```diff
@@ -15,7 +16,76 @@
         "level": {
             "value": 1
         },
-        "rules": [],
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.SkillArcana",
+                        "value": "arcana"
+                    },
+                    {
+                        "label": "PF2E.SkillAthletics",
+                        "value": "athletics"
+                    },
+                    {
+                        "label": "PF2E.SkillCrafting",
+                        "value": "crafting"
+                    },
+                    {
+                        "label": "PF2E.SkillDeception",
+                        "value": "deception"
+                    },
+                    {
+                        "label": "PF2E.SkillDiplomacy",
+                        "value": "diplomacy"
+                    },
+                    {
+                        "label": "PF2E.SkillIntimidation",
+                        "value": "intimidation"
+                    },
+                    {
+                        "label": "PF2E.SkillMedicine",
+                        "value": "medicine"
+                    },
+                    {
+                        "label": "PF2E.SkillNature",
+                        "value": "nature"
+                    },
+                    {
+                        "label": "PF2E.SkillOccultism",
+                        "value": "occultism"
+                    },
+                    {
+                        "label": "PF2E.SkillPerformance",
+                        "value": "performance"
+                    },
+                    {
+                        "label": "PF2E.SkillReligion",
+                        "value": "religion"
+                    },
+                    {
+                        "label": "PF2E.SkillSociety",
+                        "value": "society"
+                    },
+                    {
+                        "label": "PF2E.SkillSurvival",
+                        "value": "survival"
+                    },
+                    {
+                        "label": "PF2E.SkillThievery",
+                        "value": "thievery"
+                    }
+                ],
+                "flag": "skill",
+                "key": "ChoiceSet"
+            },
+            {
+                "key": "FlatModifier",
+                "label": "Skilled",
+                "selector": "{item|flags.pf2e.rulesSelections.skill}",
+                "value": "@actor.masterAbilityModifier"
+            }
```

Comment:
This should be an `AdjustModifier` changing the single modifier value rather than an untyped bonus